docker/stellargraph/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM python:3.6-slim
+
+RUN adduser --disabled-password --gecos '' stellar
+
+USER stellar
+WORKDIR /home/stellar
+
+ENV PATH=${PATH}:/home/stellar/.local/bin
+
+COPY --chown=stellar . /home/stellar/stellargraph/
```

Comment:
According to dive, this has copied 114MB of data into the image, e.g.:

- 40MB in `.git`
- 14MB in `.tox` folder (including a 3.6MB python3.6 binary!)
- 44MB in `tests/resources`

I believe these are all not truly needed (e.g. `.tox` is temporary), and all this gets deleted in the next `RUN` line.

Then again... the `RUN` line installs 900MB of packages (of which stellargraph itself is <1MB) into `~/.local/lib/python3.6/site-packages`, so maybe this doesn't matter.

Reply:
🤔 nice catch! I can do a `rm -rf` on `.tox`+`.git` in `RUN` or add them in a [.dockerignore](https://docs.docker.com/engine/reference/builder/#dockerignore-file) file

Reply:
Actually may not be needed since like you mentioned it's removed so it won't really add to the final image size

Reply:
It does add to the size of the layers, meaning the size of all the data that gets downloaded. Is it possible to somehow reuse the gitignore file as the dockerignore? The former already filters out uninteresting/temporary files, and will be kept up to date better.

Reply:
We could potentially create a symbolic link?

Reply:
It looks like the syntax is not quite a perfect match https://github.com/LinusU/gitignore-to-dockerignore

Reply:
🤔 I guess we can use that tool to autogenerate it for now